Add Date Visited

PremBagda opened the pull request:
Note changes:
EditCommandParserTest: parse_oneFieldSpecified_success() // Country Code has been commented out as it awaits merge pull request #82
EditCommandParserTest: parse_oneFieldSpecified_success() // Date Visited has been commented out as it can only execute correctly after merge pull request #82. This is to prevent merge conflict.
JsonSerializableTravelBuddyTest: toModelType_typicalPlacesFile_success() // I converted it into string comparison.

I have commented out some code written by Chung Ming in the files MainApp.java and ChartUi.java. This is because those lines prevented me from being able to launch the app.

PremBagda committed:
Add Date Visited

Note changes:
EditCommandParserTest: parse_oneFieldSpecified_success() // Country Code has been commented out as it awaits merge pull request #82
EditCommandParserTest: parse_oneFieldSpecified_success() // Date Visited has been commented out as it can only execute correctly after merge pull request #82. This is to prevent merge conflict.
JsonSerializableTravelBuddyTest: toModelType_typicalPlacesFile_success() // I converted it into string comparison.

I have commented out some code written by Chung Ming in the files MainApp.java and ChartUi.java. This is because those lines prevented me from being able to launch the app.

nivenang commented on src/test/java/seedu/travel/storage/JsonSerializableTravelBuddyTest.java:
```diff
@@ -28,9 +28,11 @@
         TravelBuddy typicalPlacesTravelBuddy = TypicalPlaces.getTypicalTravelBuddy();
-        assertEquals(travelBuddyFromFile, typicalPlacesTravelBuddy);
+
+        assertEquals(travelBuddyFromFile.toString(), typicalPlacesTravelBuddy.toString());
```
I think this is not necessary? Can just use the non-toString version if results are the same, since the toString invokes an unnecessary function call

PremBagda replied:
I tried using without toString() but it throws an error which does not make sense.
![image](https://user-images.githubusercontent.com/35837791/54875838-eb0ecd00-4e40-11e9-9f1d-056401f33683.png)

nivenang replied:
I got that error too, that's due to the TypicalPlaces.java places not matching those in typicalPlacesTravelBuddy.json file

PremBagda replied:
Thanks for the solution. I have implemented it and it works now.

nivenang reviewed (commented):
Great work! Have a question about the JsonSerializableTravelBuddyTest

PremBagda committed:
Fix bug on typicalPlacesTravelBuddy

Changed the data in typicalPlacesTravelBuddy.json to match JsonSerializableTravelBuddy. Removed toString() methods in JsonSerializableTravelBuddyTest.java

PremBagda committed:
Remove commented code related to google import statements.

chung-ming reviewed (approved):
Nice job. Will merge it to the team repo
